src/js/index.js

```diff
@@ -7,3 +7,24 @@ navigation()
 bookmark()
 formular()
 quizCardAnswer()
+
+// IDEA: randomizer button for random question
+
+const quizCard = document.querySelector('[data-js=quiz-card]')
```

Comment:
You want to have all quiz cards
```suggestion
const quizCards = document.querySelectorAll('[data-js=quiz-card]')
```